addons/danger/functions/fnc_leaderArtillery.sqf

```diff
@@ -24,11 +24,8 @@ if (_pos isEqualTo []) then {
 // check if mod active
 if (!GVAR(Loaded_WP)) exitWith {if (GVAR(debug_functions)) then {systemchat format ["%1 Artillery failed -- mod not enabled", side _unit]}};
 
-// sort target
-if (_unit distance _pos < 100) exitWith {if (GVAR(debug_functions)) then {systemchat format ["%1 Artillery failed -- target too close", side _unit]}};
-
 // settings
-private _artillery = missionNamespace getVariable ["lambs_artillery_" + str (side _unit), []];
+private _artillery = missionNamespace getVariable [QGVAR(artillery_) + str (side _unit), []];
```

Comment:
Careful, `side unit` can report back `civilian` if the unit is dead.

Reply:
this should only get called if the unit is alive because else the FSM is Termitated